adba/aniDBAbstracter.py

```diff
@@ -99,15 +100,20 @@ def add_notification(self):
 
 
 class Anime(aniDBabstractObject):
-    def __init__(self, aniDB, name=None, aid=None, tvdbid=None, paramsA=None, autoCorrectName=False, load=False):
+    def __init__(self, aniDB, name=None, aid=None, tvdbid=None, paramsA=None, autoCorrectName=False, load=False, cache_path=None):
 
         self.maper = AniDBMaper()
-        self.tvDBMap = TvDBMap()
+
+        if cache_path and not path.exists(cache_path):
+            makedirs(cache_path)
```

Comment:
What if makedirs fails? Missing permissions, etc. Graceful handling?

Reply:
How would you gracefull handle this? Show an error, and reraise the exception?
I can catch it with an OSerror. But then what? It should stop doing anything after that right?